src/plugins/intel_cpu/tests/unit/ngraph_transformations/arm/convert_group_conv.cpp

```diff
@@ -0,0 +1,94 @@
+        auto weights = ngraph::opset1::Constant::create(ngraph::element::f32, weights_shape, { 1 });
+        auto conv = std::make_shared<T>(param,
+                                        weights,
+                                        ngraph::Strides{1},
+                                        ngraph::CoordinateDiff{0},
+                                        ngraph::CoordinateDiff{0},
+                                        ngraph::Strides{1});
+
+        return std::make_shared<ngraph::Function>(ngraph::NodeVector{ conv }, ngraph::ParameterVector{ param });
+}
+
+TEST_F(TransformationTestsF, CheckConvertGroupConvIsApplied) {
+    {
+        auto param = std::make_shared<ngraph::opset1::Parameter>(ngraph::element::f32, ngraph::Shape{1, 6, 224});
+        function = createInitGraph<ngraph::opset1::GroupConvolution>(param, ngraph::Shape{2, 1, 3, 5});
+        manager.register_pass<ConvertGroupConvolution>();
+    }
+    {
+        const unsigned int groups = 2;
+        const unsigned int channel_axis = 1;
+        auto param = std::make_shared<ngraph::opset1::Parameter>(ngraph::element::f32, ngraph::Shape{1, 6, 224});
+        auto weights = ngraph::opset1::Constant::create(ngraph::element::f32, ngraph::Shape{groups, 1, 3, 5}, { 1 });
+        auto split_weights = std::make_shared<ngraph::opset1::Split>(weights,
+                                                                     ngraph::opset1::Constant::create(ngraph::element::i64, ngraph::Shape{}, {0}),
+                                                                     groups);
+        auto axis  = ngraph::opset1::Constant::create(ngraph::element::i64, ngraph::Shape{}, {channel_axis});
+        auto split = std::make_shared<ngraph::opset1::Split>(param, axis, groups);
+        ngraph::NodeVector concat_inputs;
+        for (size_t g = 0; g < groups; g++) {
+            auto out = split->output(g);
+            auto filter = std::make_shared<ngraph::opset1::Squeeze>(split_weights->output(g),
+                                                                    ngraph::opset1::Constant::create(ngraph::element::i64, ngraph::Shape{}, {0}));
+            auto conv = std::make_shared<ngraph::opset1::Convolution>(out,
+                                                                      filter,
+                                                                      ngraph::Strides{1},
+                                                                      ngraph::CoordinateDiff{0},
+                                                                      ngraph::CoordinateDiff{0},
+                                                                      ngraph::Strides{1});
+            concat_inputs.push_back(conv);
+        }
+        auto concat = std::make_shared<ov::opset8::Concat>(concat_inputs, 1);
+        function = std::make_shared<ngraph::Function>(ngraph::NodeVector{ concat }, ngraph::ParameterVector{ param });
```

Comment:
I guess it should be "function_ref", otherwise we do not actually test anything here (and some other test instances as well).
TBH, I don't really like the idea of performing the test check inside TearDown, since from the test instance perspective it is not clear, what is being tested here.
I think we should explicitly compare the functions, i.e. how it is done in convert_to_interaction.cpp tests?

Reply:
fixed